specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/storagecache.json

```diff
@@ -0,0 +1,2623 @@
+      "properties": {
+        "container": {
+          "description": "Container used for the amlFilesystem.",
+          "$ref": "#/definitions/URLString"
+        },
+        "importPrefix" : {
+          "description": "Path/Prefix for blobs in above container.  Only blobs containing this prefix get pre-hydrated into the cluster namespace",
+          "readOnly": true,
+          "type" : "string"
+        },
+        "exportPrefix" : {
+          "description": "Path/Prefix in above blob container where data gets exported when an archive task runs",
+          "type" : "string"
+        }
+      }
+    },
+    "amlFilesystemNamespaceHydrationStatus" : {
+      "description" : "Information about the amlFilesystem namespace hydration status",
+      "type" : "object",
+      "readOnly" : true,
+      "properties": {
+        "state" : {
+          "description" : "The state of the namespace hydration",
+          "type" : "string",
+          "enum" : [
+            "NotStarted",
+            "NotConfigured",
+            "InProgress",
+            "Failed",
+            "Complete"
+          ],
+          "x-ms-enum": {
+            "name": "NamespaceHydrationStatusType",
+            "modelAsString": true
+          }
+        },
+        "progress" : {
+          "description" : "Progress information for the namespace hydration",
+          "type" : "object",
+          "properties" : {
+            "blobCount" : {
+              "description" : "The number of Blobs processed.",
+              "type": "integer"
+            },
+            "totalFileSizeTiB" : {
+              "description" : "Aggregate size of blobs ingested so far in TiB",
+              "type": "number",
+              "format": "float"
+            },
+            "namesPerSec" : {
+              "type" : "integer",
+              "description": "Rate of ingest accounting for the last 30 seconds"
+            },
+            "blobName" : {
+              "type" : "string",
+              "description" : "Name of a recent blob"
+            },
+            "errorCount" : {
+              "type" : "integer",
+              "description" :"Number of failures"
+            }
+          }
+        }
+      }
+    },
+    "amlFilesystemArchiveInfo": {
+      "type":"object",
+      "description": "Information required to execute the archive operation",
+      "properties": {
+        "filesystemPath" : {
+          "type" : "string",
+          "description" : "Path of the data to archive relative to the root of the Lustre file system.  Specify '/' to archive all modified data."
+        }
+      }
+    },
+    "amlFilesystemArchive" : {
+      "description" : "Information about the amlFilesystem archive",
+      "type" : "object",
+      "readOnly": true,
+      "properties": {
+        "filesystemPath" : {
+          "readOnly": true,
+          "type" : "string",
+          "description" : "The path to the filesystem to archive."
```

Comment:
This must match the filesystemPath description under amlFilesystemArchiveInfo

Reply:
Fixed in next update.